src/fireblocks-sdk.ts

```diff
@@ -1148,19 +1152,58 @@ export class FireblocksSDK {
         return await this.apiClient.issueDeleteRequest(`/v1/fee_payer/${baseAsset}`);
     }
 
-    /**
-     * Get all signer connections of the current user
-     * @returns Array of sessions
-     */
-    public async getAllSignerConnections(): Promise<Session[]> {
-        return await this.apiClient.issueGetRequest(`/v1/connections`);
+    private getWeb3ConnectionPath(type: Web3ConnectionType) {
+        const basePath = `/v1/connections`;
+
+        switch (type) {
+            case(Web3ConnectionType.WALLET_CONNECT): {
+                return `${basePath}/wc`;
+            }
+            default: {
+                return basePath;
```

Comment:
returning the default is valid from the server side?

Reply:
Yeah it also routes to the service, will just be 404 for those methods

Reply:
Changed to throw error